src/app/logic/localStorage.js

```diff
@@ -0,0 +1,23 @@
+function addResult(rankings, gameInfo, amountOfResults) {
+  const { name, result, mode } = gameInfo;
+  // push gameinfo (only name and result, we don't need mode in this place) into the right mode in
+  // rankings
+  rankings[mode].push({ name, result });
+  // sort in descending order by points
+  rankings[mode].sort((a, b) => (a.result < b.result ? 1 : -1));
+  // remove last element if there's to many gameinfos in our game ranking
+  if (rankings[mode].length > amountOfResults) rankings[mode].pop();
+  // update rankings in localStorage
+  localStorage.setItem('rankings', JSON.stringify(rankings));
+}
+
+/**
```

Comment:
It's a great idea to add description! Will make it so much easier for the next person working on this code. Maybe you can follow the jsdoc pattern and place it above the function for better understanding. https://jsdoc.app/about-getting-started.html